App/Redux/TextileRedux.js

```diff
@@ -36,10 +38,13 @@ export const INITIAL_STATE = Immutable({
   devices: []
 })
 
-/* ------------- Selectors ------------- */
 
+/* ------------- Selectors ------------- */
 export const TextileSelectors = {
   // TODO: Add more selectors here as we learn how they are used
+  items: state => {
```

Comment:
Was thinking this could be a function like: ```
getPathForUploadId: (state, uploadId) => {
  // Do all the filtering of the state in here to find the image based on the uploadId and return the file path
}
```